src/utilities/notifications.ts

```diff
@@ -20,3 +43,75 @@ export function getPostUpdatedNotificationContent( userState: UserState, status:
 
 	return `${ noun } ${ verb } by ${ userState.name }.`;
 }
+
+/**
+ * Get the content of a user presence notification, based on the type.
+ *
+ * @param userState the user state of the user related to the notification
+ * @param type the type of notification
+ * @returns the content of the user presence notification
+ */
+export function getUserPresenceNotificationContent(
+	userState: UserState,
+	type: NotificationType
+): string {
+	let action = type === NotificationType.UserEntered ? 'entered' : 'exited';
+	return `${ userState.name } has ${ action } the post.`;
+}
+
+function shouldSendNotification( userState: UserState, type: NotificationType ): boolean {
```

Comment:
This is the notification skipping code I've added. I think this is better suited to be here, so we can simplify the actual usage of these methods.